src/renderer/Renderer.ts

```diff
@@ -226,10 +226,17 @@ abstract class Renderer {
     const flicking = getFlickingAttached(this._flicking, "Renderer");
 
     const cameraElement = flicking.camera.element;
-    const cameraChilds = toArray(cameraElement.children);
+    const cameraChilds = toArray(cameraElement.childNodes);
 
     const align = this._getPanelAlign();
 
+    // Remove all text nodes
+    cameraChilds.forEach(child => {
+      if (child.nodeType === Node.TEXT_NODE) {
```

Comment:
Perhaps it is text, comment, element that can be childNodes.

Since childNodes become panels, I think it's good to remove everything except elements.

Reply:
That's a good point, but does the comment node has a size(width) too?

Reply:
It seems that the comment node has no width.

Will the code that deletes the text node run in angular?

If so, it seems to be a problem.

Reply:
Well, actually you're seeing an outdated version of the Renderer.
The newer version uses `ElementManipulator` to do that, which isn't included in the Angular.

Reply:
Good. I think there's no problem if you use `children`.